templates/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Registration Form</title>
+    <!-- Вставьте любые стили или скрипты, которые вам нужны -->
+</head>
+<body>
+
+<h1>Registration Form</h1>
+

```

Comment:
Обработку flash сообщений можно добавить тут
```suggestion
    {% with messages = get_flashed_messages() %}
        {% if messages %}
        <ul>
            {% for message in messages %}
            <li>
                    {{ message }}
            </li>
            {% endfor %}
        </ul>
        {% endif %}
    {% endwith %}
```